MAUI/MauiAppB2C/MSALClient/PCAWrapperB2C.cs

```diff
@@ -2,17 +2,24 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Diagnostics;
 using Microsoft.Identity.Client;
 
 namespace MauiB2C.MSALClient
 {
+    /// <summary>
+    /// This is a wrapper for PublicClientApplication. It is singleton and can be utilized by both application and the MAM callback
+    /// </summary>    
     public class PCAWrapperB2C
     {
         /// <summary>
-        /// This is the singleton used by consumers
+        /// This is the singleton used by consumers. Since PCAWrapper constructor does not have perf or memory issue, it is instantiated directly.

```

Comment:
Why would we ever publish code that has perf or memory issue?

Reply:
This is in reference to your comment here  https://github.com/Azure-Samples/active-directory-xamarin-native-v2/pull/203#discussion_r945597880